### ISSUE TITLE
Quotation marks don't work for Windows

### ISSUE THREAD

**Issue by vsajip** [vsajip/sarge#52]
**[Original report](https://bitbucket.org/vinay.sajip/sarge/issue/52) by Anonymous.**

----------------------------------------

sarge.run('git rev-list origin/master --since="1 hours ago"')

it returns 128 error because it does not see "" for "1 hours ago"
How I can avoid this bug for Windows?

**Comment by vsajip** [comment]
What happens if you do the following?

```python
sarge.run('git rev-list origin/master --since="1 hours ago"', posix=True)
```

‌

**Comment by vsajip** [comment]
I've [updated](https://docs.red-dove.com/sarge/tutorial.html#parsing-commands-under-windows) the documentation.

**Comment by vsajip** [comment]
**Original changes by Vinay Sajip (Bitbucket: [vinay.sajip](https://bitbucket.org/vinay.sajip), GitHub: [vsajip](https://github.com/vsajip)).**

----------------------------------------

changed state from "new" to "resolved"

**Comment by vsajip** [comment]
Fix #52: Update documentation related to parsing on Windows.
